prometheus.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 func processWriteRequest(req *prompb.WriteRequest) ([][]byte, error) {
+	logrus.WithField("var", req).Debugln()
```

Comment:
Do you really want to keep this log? It can be too verbose. Is there anything such as `Traceln`?

Reply:
No, `Debug` is the lower level available. However I'm forcing `Info` as default. https://github.com/Telefonica/prometheus-kafka-adapter/blob/a19e444393114051bff0c323b781ace10147a7a8/config.go#L59-L60